CRM/Contact/BAO/Query.php

```diff
@@ -5008,9 +5008,7 @@ public function alphabetQuery() {
     $query = "SELECT DISTINCT LEFT(contact_a.sort_name, 1) as sort_name
       {$sqlParts['from']}
       {$sqlParts['where']}
-      {$sqlParts['having']}
-      GROUP BY sort_name
-      ORDER BY sort_name asc";
+      {$sqlParts['having']}";
```

Comment:
Note it's invalid to have a having without a group by - however, it's also invalid to have one after a group by I think so ??? (WTF)